bedrock/base/templates/base-pebbles.html

```diff
@@ -102,9 +102,15 @@
             <li class="first about-item">
               <a href="{{ url('mozorg.about') }}" data-link-type="nav" data-link-name="About">{{_('About')}}</a>
             </li>
+            {% if LANG.startswith('en-') %}
+            <li class="technology-item">
+              <a href="{{ url('mozorg.technology') }}" data-link-type="nav" data-link-name="Web Innovations" data-alt-text="{{ _('Innovations') }}">{{ _('Web Innovations') }}</a>
```

Comment:
I included this as I was worried that "Web Innovations" may well be too long to fit in the nav for some locales at smaller screen widths. If you think I'm being over cautious here I can be persuaded to drop it?